layersvt/device_simulation.cpp

```diff
@@ -1247,6 +1247,11 @@ VKAPI_ATTR void VKAPI_CALL GetPhysicalDeviceProperties(VkPhysicalDevice physical
 
 VKAPI_ATTR void VKAPI_CALL GetPhysicalDeviceProperties2KHR(VkPhysicalDevice physicalDevice,
```

Comment:
This commit seems incomplete compared to the GetPhysicalDeviceFeatures2KHR replacement in the previous commit.  There is not a GetPhysicalDeviceProperties2 entry point, that would also be added to getprocaddress.